src/prefect/environments/execution/dask/remote.py

```diff
@@ -49,7 +49,7 @@ class RemoteDaskEnvironment(RemoteEnvironment):
     def __init__(
         self,
         address: str,
-        security: Type[Security] = None,
+        security: Security = None,
```

Comment:
As you no longer use `typing.Type` the static analysis check is failing with an usused import